src/integer/mat_z/get.rs

```diff
@@ -252,6 +312,49 @@ mod test_get_num {
     }
 }
 
+#[cfg(test)]
+mod test_get_vec {
+
+    use crate::integer::MatZ;
+    use std::str::FromStr;
+
+    #[test]
+    fn get_vec_row_works() {
+        let matrix = MatZ::from_str(&format!("[[1,2,3],[4,{},{}]]", i64::MAX, i64::MIN)).unwrap();
+        let row = matrix.get_vec_row(1).unwrap();
```

Comment:
```suggestion
        let matrix = MatZ::from_str(&format!("[[0,0,0][1,2,3],[4,{},{}]]", i64::MAX, i64::MIN)).unwrap();
        let row_0 = matrix.get_row(0).unwrap();
        let row_1 = matrix.get_row(1).unwrap();
        let row_2 = matrix.get_row(2).unwrap();
```
Please test all rows that you initialized and also include a zero row/column to the tests and compare it appropriately. I shortly took a look whether a check if the row/column is zero is worth it, but I believe that the check takes up almost as much time as fetching and setting the entire row (and getting a zero vector should hopefully not be done too often).